src/Package.vala

```diff
@@ -282,19 +282,19 @@ public class Eddy.Package : Object {
 
         int length = int.max (aparts.length, bparts.length);
```

Comment:
this is not a problem coincidentally because you have `return 0`, but if you do it correctly it will cause out of array error, replace it with `int.min` and do the length comparison as above